.github/workflows/release.yml

```diff
@@ -32,9 +32,26 @@ jobs:
     outputs:
       hasWritePermission: ${{ steps.check.outputs.require-result }}
 
+  build-master:
+    name: Build master
+    needs: check-permission
+    if: contains(needs.check-permission.outputs.hasWritePermission, 'true')
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@master
+        with:
+          fetch-depth: 0
+          repository: ${{ github.repository }}
+          ref: master
+      - name: Build and validate
+        run: |
+          ./validate.sh
+          go build .
```

Comment:
I am not sure we need the `go build .` I don't think it is possible for it not to compile, but still manage to pass all the unit tests,

Reply:
Isn't this the build which produces the actual binaries? If we don't call it, we don't have an app?

Reply:
Agree `go build . ` command will execute successfully only if all tests pass.  Can update script to only run validate.sh

```
       - name:  Validate 
         run: ./validate.sh
         
```

Reply:
Good catch. Removed